torch/csrc/generic/Storage.cpp

```diff
@@ -319,7 +319,6 @@ typedef PyObject *(*getter)(PyObject *, void *);
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays,cppcoreguidelines-avoid-non-const-global-variables)
 static struct PyGetSetDef THPStorage_(properties)[] = {
   {"device", (getter)THPStorage_(device), nullptr, nullptr, nullptr},
-  {"dtype",  (getter)THPStorage_(dtype), nullptr, nullptr, nullptr},
```

Comment:
This can be later, but one consequence of untyped Storage is I hope we can remove Storage.cpp and related files out of generic/ (which is still doing old style TH macro substitution)